browser-test/src/support/applicant_questions.ts

```diff
@@ -527,8 +532,16 @@ export class ApplicantQuestions {
     }
   }
 
-  async expectIneligiblePage() {
-    expect(await this.page.innerText('html')).toContain('you may not qualify')
+  async expectIneligiblePage(northStar = false) {
+    if (northStar) {
+      await expect(
+        this.page.locator('text="You may not be eligible for this program"'),
+      ).toBeVisible()
+      await expect(this.page.locator('#another-program')).toBeVisible()
```

Comment:
Optional: I assume this is done this way in other places too, but in general we should avoid locating by ids when possible and use visible text or roles.